package.json

```diff
@@ -2,6 +2,13 @@
   "name": "react-starter",
   "version": "0.1.0",
   "private": true,
+  "scripts": {
```

Comment:
I would rather we use something standard in create-react-app to handle relative pathing rather than adding craco.  in my last project we used this:
```
    "paths": {
      "@/*": ["src/*"]
    },
```
This may not cover it all try googling (example: https://dev.to/smitterhane/avoid-relative-path-import-hell-in-react-36in).